server/service/integration_core_test.go

```diff
@@ -642,7 +642,7 @@ func (s *integrationTestSuite) TestListHosts() {
 	host.HostSoftware = fleet.HostSoftware{
 		Modified: true,
 		Software: []fleet.Software{
-			{Name: "foo", Version: "0.0.2", Source: "chrome_extensions"},
+			{Name: "foo", Version: "0.0.1", Source: "chrome_extensions"},
```

Comment:
Why this change?

Reply:
Because we are not deleting software when we delete a host, so there's a test that fails if you run the whole integration tests, even though we are cleaning the hosts.

Reply:
Ah, a conflict, gotcha.